plugins/oauth/hooks.js

```diff
@@ -16,26 +13,9 @@ let SDK = require('./SDK').getSDK();
 let {AccessDeniedPage, Authenticated, ConfigStore, CookieUtils} = SDK.get([
   'AccessDeniedPage', 'Authenticated', 'ConfigStore', 'CookieUtils']);
 
-// DCOS-5935 Build mixin for non-react components to listen to store events
-const CONFIG_STORE_LISTENERS = [CONFIG_LOADED, CONFIG_ERROR];
-
-function reloadConfigStore(callback) {
-  let handler = function () {
-    CONFIG_STORE_LISTENERS.forEach(event => {
-      ConfigStore.removeChangeListener(event, handler);
-    });
-
-    callback();
-  };
-
-  CONFIG_STORE_LISTENERS.forEach(event => {
-    ConfigStore.addChangeListener(event, handler);
-  });
-
-  ConfigStore.fetchConfig();
-}
+let configResponseCallback = null;
 
-module.exports = {
+module.exports = _.extend({
```

Comment:
it would probably also be easier to read if you did `Object.assign({}, StoreMixin, {other,stuff})`
Rather than having `StoreMixin` at the bottom of the page.